mindmap/static/html/mindmap.html

```diff
@@ -1,5 +1,8 @@
-<div class="mindmap_block">
-<p>MindMapXBlock: count is now
-    <span class='count'>{self.count}</span> (click me to increment).
-</p>
-</div>
+<head>
+    <link type="text/css" rel="stylesheet" href="https://unpkg.com/jsmind@0.6.4/style/jsmind.css" />
+    <script type="text/javascript" src="https://unpkg.com/jsmind@0.6.4/es6/jsmind.js"></script>
+    <script type="text/javascript" src="https://unpkg.com/jsmind@0.6.4/es6/jsmind.draggable-node.js"></script>
```

Comment:
Could you download this cdn's?  if the unpks's server go down they won't be available

Reply:
Done